ultimaker_tough_pla_blue.xml.fdm_material

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
+    <metadata>
+        <name>
+            <brand>Ultimaker</brand>
+            <material>Tough PLA</material>
+            <color>Blue</color>
+        </name>
+        <GUID>4b049931-6ee9-408c-8588-ddd4673467d1</GUID>
+        <version>22</version>
```

Comment:
Shouldn't all of these new profiles be `<version>1</version>`?

Reply:
These are just new colours of existing material (UM Tough PLA), hence the same version number as other Tough PLAs, basically what Carlo also says above.

Reply:
[ Suggestion ]
I agree with @atrebelo; would suggest a v1, since it's the first version of the profile for this material.
I do not see the benefits, can you maybe elaborate where starting at 22 helps out? @ckielstra / @Andi-ultimaker ?

No biggy, but if it doesn't matter, why not set it to 1 then :P

Reply:
Yeah, I'm also questioning why not just use 1 since they don't already exist 🤷🏻‍♀️

Reply:
Just wanted to keep it consistent with existing Tough PLA colours.

Reply:
Ok I see, then if that's what the version number stands for, it'd be misleading to keep the same one as the existing colours.